src/app/content/highlights/components/EditCard.tsx

```diff
@@ -69,6 +71,16 @@ const EditCard = React.forwardRef<HTMLElement, Props>((
     }
   };
 
+  React.useEffect(() => {
+    if (data) { return; }
+    if (authenticated) {
+      trackShowCreate();
+    } else {
+      trackShowLogin();
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [data]);
```

Comment:
this'll make it re-run when data is changed. not that that probably matters but you might as well leave it out and then it'll only run on mount